thavalon-webapp/src/components/gameComponents/playerBoard.tsx

```diff
@@ -50,9 +37,32 @@ export function PlayerBoard(): JSX.Element {
     // State for maintaining the player list.
     const [playerList, setPlayerList] = useState<string[]>([])
     // State maintaining selected players. These players are highlighted in green.
```

Comment:
or highlighted in red if mission 1 second proposal, right? Can you add comment clarifying diff between primary and secondary players?

Reply:
Fixed